strongbox-web-core/src/main/java/org/carlspring/strongbox/controllers/configuration/StoragesConfigurationController.java

```diff
@@ -0,0 +1,297 @@
+    public ResponseEntity getRepository(@ApiParam(value = "The storageId", required = true)
+                                        @PathVariable final String storageId,
+                                        @ApiParam(value = "The repositoryId", required = true)
+                                        @PathVariable final String repositoryId)
+    {
+
+        try
+        {
+            Repository repository = configurationManagementService.getStorage(storageId)
+                                                                  .getRepository(repositoryId);
+
+            if (repository != null)
+            {
+                return ResponseEntity.status(HttpStatus.OK)
+                                     .body(repository);
+            }
+            else
+            {
+                return ResponseEntity.status(HttpStatus.NOT_FOUND)
+                                     .body("Repository " + storageId + ":" + repositoryId + " was not found.");
+            }
+        }
+        catch (Exception e)
+        {
+            logger.error(e.getMessage(), e);
+
+            return ResponseEntity.status(HttpStatus.NOT_FOUND)
+                                 .body("Repository " + storageId + ":" + repositoryId + " was not found.");
+        }
+    }
+
+    @ApiOperation(value = "Deletes a repository.")
+    @ApiResponses(value = { @ApiResponse(code = 200,
+                                         message = "The repository was deleted successfully."),
+                            @ApiResponse(code = 404,
+                                         message = "Repository ${storageId}:${repositoryId} was not found!"),
+                            @ApiResponse(code = 500,
+                                         message = "Failed to remove repository ${repositoryId}!") })
+    @PreAuthorize("hasAuthority('CONFIGURATION_DELETE_REPOSITORY')")
+    @RequestMapping(value = "/{storageId}/{repositoryId}",
+                    method = RequestMethod.DELETE,
+                    produces = MediaType.TEXT_PLAIN_VALUE)
+    public ResponseEntity removeRepository(@ApiParam(value = "The storageId", required = true)
+                                           @PathVariable final String storageId,
+                                           @ApiParam(value = "The repositoryId", required = true)
+                                           @PathVariable final String repositoryId,
+                                           @ApiParam(value = "Whether to force delete the repository from the file system")
+                                           @RequestParam(name = "force", defaultValue = "false") final boolean force)
+    {
+        final Repository repository = configurationManagementService.getStorage(storageId)
+                                                                    .getRepository(repositoryId);
+        if (repository != null)
+        {
+            try
+            {
+                logger.debug(storageId);
```

Comment:
Could you please remove these two debug statements? They're useless as they are.